vector-stores/spring-ai-milvus-store/pom.xml

```diff
@@ -54,6 +54,15 @@
 			<version>${milvus.version}</version>
 		</dependency>
 
+		<!-- This is HORRIBLE! But IntelliJ doesn't want to work without it -->
```

Comment:
There's something in the Milvus Java SDK causing a compilation error in IntelliJ if Lombok is not added as a dependency. And the error propagates to the spring-ai-spring-boot-auto-configure module, making it impossible to run auto tests from the IDE. I should probably remove this from the PR (I forgot to undo the change before committing), but since it's here I figured it'd be good to bring this up.